lib/matplotlib/container.py

```diff
@@ -114,6 +114,24 @@ def __init__(self, patches, errorbar=None, **kwargs):
 
 
 class ErrorbarContainer(Container):
+    '''
+    Container for errobars.
+
+    Atributes
```

Comment:
This didn't render:  https://450-74355158-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/container_api.html#matplotlib.container.ErrorbarContainer

Attributes?

Reply:
🤦‍ thanks!